stowdir/.local/bin/csvstats

```diff
@@ -268,6 +236,19 @@ def main(args):
     print("mean:", stats.mean, file=args.output)
     print("stddev:", stats.stddev, file=args.output)
 
+    if args.histogram is not False:
+        import matplotlib.pyplot as plt
+        import seaborn as sns
+
+        bins = "auto" if args.histogram is None else args.histogram
+
+        sns.set_theme()
+        ax = sns.histplot(values, kde=True, bins=bins)
```

Comment:
This wasn't as useful as I had hoped to troubleshoot a particular issue I'm facing at work.

I think this may need to be more configurable. I had trouble getting a reasonable histogram out of ~30k samples with really consistent deltas.

Perhaps a histogram isn't the right format, or perhaps I should normalize / log-transform the input data?

Reply:
![image](https://github.com/Notgnoshi/dotfiles/assets/8878460/7f58507e-3814-4bd5-87ae-1479ecde2f67)

Adding `--range` helped significantly.